.pre-commit-config.yaml

```diff
@@ -17,7 +17,7 @@ repos:
     hooks:
       - id: yamllint
   - repo: https://github.com/awslabs/cfn-python-lint
-    rev: v1.2.5a9
+    rev: v0.86.4
```

Comment:
These changes are necessary from this PR https://github.com/Sceptre/sceptre/pull/1470 I merged that branch in here so that the build could be green.